### PR TITLE
Update webpack and other packages

### PR DESCRIPTION
**Pull request by PiotrKozlowski**
Everything should work the same as before.

I didn't `eject` webpack scripts because this is why we had to update this whole thing manually - there is no easy way to update webpack breaking changes in projects which ejected scripts.

Checks for missing l10n keys from `FtlPlugin.js` will be recreated in #280

### CODE REVIEW

**Comment by MartaSztuka** [comment]
Everything work as before

File: src/containers/Chat/components/Message/index.tsx [hunk 1]
```diff
@@ -152,7 +152,7 @@ function Line({ data }: { data: Data }): JSX.Element[] {
     case FRAME_HYPERLINK: {
       const [len, inx] = leb128(body, 0)
       const label = decode_utf8(body.subarray(inx, inx + len))
-      let url = String.fromCodePoint.apply(null, body.subarray(inx + len))
```
Comment (aiwenar) [inline src/containers/Chat/components/Message/index.tsx]: This introduces an unnecessary copy.

File: src/helpers/saveAsFile.ts [hunk 2]
```diff
@@ -1,14 +1,4 @@
...
-  } else {
-    const elem = window.document.createElement('a')
-    elem.href = window.URL.createObjectURL(blob)
-    elem.download = filename
-    document.body.appendChild(elem)
-    elem.click()
-    window.URL.revokeObjectURL(elem.href)
-    document.body.removeChild(elem)
-  }
+  window.navigator.msSaveBlob(blob, filename)
```
Comment (aiwenar) [inline src/helpers/saveAsFile.ts]: What is this function? MDN doesn't know it.

File: src/screens/app/Draft/components/LocalizationLoader/index.tsx [hunk 3]
```diff
@@ -59,7 +59,7 @@ class LocalizationLoader extends React.Component<LocalizationLoaderProps> {
   }
 }
 
-export default Load(loader, ['locale'], undefined, undefined, EditorLoader)<{}>(LocalizationLoader)
+export default Load(loader, ['locale'], undefined, undefined, EditorLoader)<any>(LocalizationLoader)
```
Comment (aiwenar) [inline src/screens/app/Draft/components/LocalizationLoader/index.tsx]:
```suggestion
export default Load(loader, ['locale'], undefined, undefined, EditorLoader)<{ locale: string, children: React.ReactNode }>(LocalizationLoader)
```

File: src/containers/Chat/components/Message/index.tsx (hunk 1 above)
Comment (PiotrKozlowski) [reply src/containers/Chat/components/Message/index.tsx]: Huh, so typescript says that `window.navigator.msSaveOrOpenBlob` is always declared so I've removed check, but now I see that this feature is not in a standard: https://developer.mozilla.org/en-US/docs/Web/API/Navigator/msSaveBlob
I'll recreate previous implementation with small fix

File: src/helpers/saveAsFile.ts (hunk 2 above)
Comment (PiotrKozlowski) [reply src/helpers/saveAsFile.ts]: Huh, so typescript says that window.navigator.msSaveOrOpenBlob is always declared so I've removed check, but now I see that this feature is not in a standard: https://developer.mozilla.org/en-US/docs/Web/API/Navigator/msSaveBlob
I'll recreate previous implementation with small fix

**Pull request merged**